core/src/main/java/org/ehcache/spi/cache/tiering/LowerCachingTier.java

```diff
@@ -0,0 +1,87 @@
+/*
+ * Copyright Terracotta, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.ehcache.spi.cache.tiering;
+
+import org.ehcache.exceptions.CacheAccessException;
+import org.ehcache.function.Function;
+import org.ehcache.function.NullaryFunction;
+import org.ehcache.spi.cache.ConfigurationChangeSupport;
+import org.ehcache.spi.cache.Store;
+import org.ehcache.spi.service.Service;
+import org.ehcache.spi.service.ServiceConfiguration;
+
+/**
+ * Lower caching tier, according to Montreal design.
+ *
+ * @author Ludovic Orban
+ */
+public interface LowerCachingTier<K, V> extends ConfigurationChangeSupport {
```

Comment:
It could, but I think Louis wanted them to be strictly separated. And I cannot remember what his arguments were... :(